chart/scripts/post-delete-hook.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+succeeded=()
```

Comment:
Need to add a check to ensure that the expected env vars exists. Example:

```suggestion
if [ -z "${namespaces}" ]; then
  echo "No namespaces provided."
  exit 0
fi

succeeded=()
```

Reply:
fixed.